general_rules/Scenario.tex

```diff
@@ -1,202 +1,122 @@
-\end{enumerate}
+	\item \textbf{Appearance:} Floor and walls are mainly uni-colored but can contain texture, e.g., a carpet on the floor, a poster or picture on the wall.\\
+\end{itemize}
 
 
 \subsection{Furniture}
-\label{rule:scenario_furniture}
-The arena will be equipped with typical objects (furniture) that are not specified in quantity and kind.
+\label{sec:rules:scenario:furniture}
+The arena will be furnished with items common in the host country.
 
 The minimal configuration consists of:
 \begin{itemize}
-	\item a bed,
-	\item a couch,
-	\item a small table,
-	\item a small dinner table with two chairs,
-	\item two trash bins,
-	\item an open cupboard or small table with a television and remote control,
-	\item a cupboard with drawers, and
-	\item a bookcase or shelf with doors and some books inside
+	\item Bed,
+	\item Couch
+	\item Small Table
+	\item Small Dinner Table with Two Chairs
+	\item Two Trash Bins
+	\item Television with Remote Control
+	\item Cupboard with Drawers
+	\item Bookcase
+	\item Coat Rack
 \end{itemize}
 
-Likewise the arena's kitchen must have:
+The \Arena\textit{'s} kitchen must have:
 \begin{itemize}
-	\item a dishwasher,
-	\item a microwave,
-	\item a sink, and
-	\item a refrigerator in the kitchen (with some cans and plastic bottles inside).
+	\item Dishwasher
+	\item Sink
+	\item Refrigerator (with some cans and plastic bottles inside).
 \end{itemize}
 
 A typical arena setup is shown in~\reffig{fig:scenario_arena}.
 
-\begin{figure}[tbp]
+\begin{figure}[H]
 	\centering
 	\subfloat[Typical arena]{\label{fig:scenario_arena}\includegraphics[height=46mm]{images/typical_arena.jpg}} ~
 	\subfloat[Typical objects]{\label{fig:scenario_objects}\includegraphics[height=46mm]{images/typical_objects.jpg}}
-	\caption{Scenario examples: (a) a typical arena, and (b) typical objects.}
+	\caption{Example \RoboCup\AtHome{} scenario.}
 	\label{fig:arena}
 \end{figure}
 
-
-\subsubsection{Cupboard}
-The cupboard can be any shelf-like furniture in which objects can be placed.
-\begin{itemize}
-	\item[\textbf{Doors:}] The cupboard may have doors.
-	\item[\textbf{Drawers:}] The cupboard must have at least two drawers betweem 90cm and 120cm from floor level.
-	\item[\textbf{Shelves:}] The minimum distance between shelf or layers is 30cm.
-\end{itemize}
-
-\subsubsection{Shelf}
-A shelf, rack, or bookcase is required in RoboCup@Home.
-The shelf can be any shelf-like furniture in which objects can be placed.
-\begin{itemize}
-	\item[\textbf{Doors:}] The shelf must have at least one door (preferrably a vertical one) covering up to one half of it.
-	\item[\textbf{Drawers:}] The shelf must have no drawers.
-	\item[\textbf{Shelves:}] The shelf must have 5 shelves or layers between 0.0m and 1.80m from the ground, with a minimum distance of 30cm between shelves or layers.
-\end{itemize}
-
-\subsubsection{Fridge}
-Fridges must not be smaller than 120m. At least one powered and functioning fridge is required.
```

Comment:
I think both @dirkholz and @robotijn will agree that at least one powered and functioning fridge is extremely important to RoboCup@Home and one of the features of the league which helps retain the committee members.

It's required.